theia/api/views/imagery_request_view_set.py

```diff
@@ -11,14 +16,37 @@ class ImageryRequestViewSet(viewsets.ModelViewSet):
     serializer_class = ImageryRequestSerializer
 
     def create(self, request, *args, **kwargs):
-        copy = request.data.copy()
-        copy['bearer_token'] = request.session['bearer_token']
-        copy['refresh_token'] = request.session['refresh_token']
-        copy['bearer_expiry'] = request.session['bearer_expiry']
-
-        serializer = self.get_serializer(data=copy)
+        serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
-        self.perform_create(serializer)
 
-        headers = self.get_success_headers(serializer.data)
-        return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
\ No newline at end of file
+        with get_authenticated_panoptes(
+                request.session['bearer_token'],
+                request.session['refresh_token'],
+                request.session['bearer_expiry']):
+            try:
+                Project.find(_id_for(request.data['project']))
+                self.perform_create(serializer)
+
+                headers = self.get_success_headers(serializer.data)
+                return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
+            except PanoptesAPIException:
+                return Response(serializer.data, status=status.HTTP_401_UNAUTHORIZED, headers=headers)
+
+
+def get_authenticated_panoptes(bearer_token, refresh_token, bearer_expiry):
+    authenticated_panoptes = Panoptes(
```

Comment:
Maybe clarify that this is using the supplied user/client credentials rather than server side ones
```suggestion
    user_authenticated_panoptes = Panoptes(
```

Reply:
Renamed!